gammapy/scripts/tests/test_cta_sensitivity.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.tests.helper import assert_quantity_allclose
+from numpy.testing import assert_allclose
+from astropy.units import Quantity
+from gammapy.utils.testing import requires_data, requires_dependency
```

Comment:
Use relative imports within Gammapy (even from the tests) for consistency with the rest of the codebase.